experimental/plugins_network.md

```diff
@@ -0,0 +1,43 @@
+# Experimental: Docker network driver plugins
+
+Docker supports network driver plugins via 
+[LibNetwork](https://github.com/docker/libnetwork). Network driver plugins are 
+implemented as "remote drivers" for LibNetwork, which shares plugin 
+infrastructure with Docker. In effect this means that network driver plugins 
+are activated in the same way as other plugins, and use the same kind of 
+protocol.
+
+## Using network driver plugins
+
+The means of installing and running a network driver plugin will depend on the
+particular plugin.
+
+Once running however, network driver plugins are used just like the built-in
+network drivers: by being mentioned as a driver in network-oriented Docker
+commands. For example,
+
+    docker network create -d weave mynet
```

Comment:
Since there are multiple plugin providers working on network plugins, maybe its a good idea to mention them as well somehow ?

Just a suggestion, we can change this to `docker network create -d <driver> mynet` and enumerate the list of plugins being developed and the status of each of them ?